maskhit/train.py

```diff
@@ -152,6 +204,10 @@ def prepare_data(meta_split, meta_file, round_month=False, vars_to_include=[]):
     elif args.outcome_type == 'survival':
         meta_split = meta_split.loc[~meta_split.status.isna()
                                     & ~meta_split.time.isna()]
+    
+    print("Printing pre-processed dataset")

```

Comment:
Please add a print tag.

Reply:
Sorry. I'm not quite sure what a print tag is